### PR TITLE
Dgds add chasm data

### PR DESCRIPTION
**Pull request by ckrogers**

### CODE REVIEW

File: hydroengine_service/datasets/dataset_visualization_parameters.json [hunk 1]
```diff
@@ -171,5 +171,52 @@
...
+            "turbulence": 0.0
+		},
+		"max": {
+			"significant_wave_height": 3.0,
+			"wave_direction": 360.0,
+            "wave_period": 9.0,
+            "turbulence": 0.5
+		},
+		"palette": {
+			"significant_wave_height": ["#042333", "#3c0912", "#730e27", "#a62225", "#c0583b", "#d08b73", "#dfbcb0", "#f1edec"],
```
Comment (SiggyF) [inline hydroengine_service/datasets/dataset_visualization_parameters.json]: from cmocean?

File: hydroengine_service/dgds_functions.py [hunk 2]
```diff
@@ -259,6 +259,13 @@ def apply_image_operation(image, operation, data_params=None, band=None):
     if operation == "magnitude":
         image = image.pow(2).reduce(ee.Reducer.sum()).sqrt().rename('magnitude')
     if operation == "flowmap":
+        # if band.contains("speed"):
```
Comment (SiggyF) [inline hydroengine_service/dgds_functions.py]: remove?

File: hydroengine_service/main.py [hunk 3]
```diff
@@ -1498,6 +1498,44 @@ def get_gebco_data():
...
+    end_date = r.get('endDate', None)
+    image_num_limit = r.get('limit', None)
+
+    if not (dataset or image_id):
+        msg = f'dataset or imageId required.'
+        logger.error(msg)
+        raise error_handler.InvalidUsage(msg)
+    if dataset:
+        source = 'projects/dgds-gee/chasm/' + dataset
+    if image_id:
```
Comment (SiggyF) [inline hydroengine_service/main.py]: elif?

File: hydroengine_service/datasets/dataset_visualization_parameters.json (hunk 1 above)
Comment (ckrogers) [reply hydroengine_service/datasets/dataset_visualization_parameters.json]: Yes, and same palette we use for other wave data.

File: hydroengine_service/main.py (hunk 3 above)
Comment (ckrogers) [reply hydroengine_service/main.py]: This has been updated in latest commit.

**Pull request merged**